src/assets/v1.4.1/compatibility_fallback_handler.json

```diff
@@ -26,6 +26,7 @@
     "17000": "0xfd0732Dc9E303f09fCEf3a7388Ad10A83459Ec99",
     "42161": "0xfd0732Dc9E303f09fCEf3a7388Ad10A83459Ec99",
     "42220": "0xfd0732Dc9E303f09fCEf3a7388Ad10A83459Ec99",
+    "444444": "0xfd0732Dc9E303f09fCEf3a7388Ad10A83459Ec99",
```

Comment:
Could you please maintain a proper ascending order? thanks

Reply:
fixed